docker/README.md

```diff
@@ -0,0 +1,129 @@
+
+> If you've already set up the dev environment, any make command containing `init-dev` will wipe all persistent data from the db and elasticsearch.
+
+> If you don't need test accounts and resources, run `make init-dev-clean` instead of the make command, below. This will save several minutes on container startup time.
+
+After setting up the workspace, you can start and stop the containers with [these commands](#start-and-stop-the-dev-environment).
+
+Run Docker, then paste the following code block into your terminal after `cd`ing to an appropriate directory:
+
+```sh
+# create the workspace
+mkdir fpan-workspace && cd fpan-workspace
+
+# clone our Arches 6.2 fork
+git clone \
+  --branch=dev/6.2.x-hms-cli \
+  --single-branch \
+  https://github.com/legiongis/arches
+
+# clone our Arches extensions
+git clone https://github.com/legiongis/arches-extensions
+cd arches-extensions
+git checkout e349934
+cd ..
+
+# clone this project and copy the Docker and files and local settings
+git clone https://github.com/legiongis/fpan
+cp ./fpan/docker/* .
+mv ./settings_local.py fpan/fpan/
+cp ./edit_dot_env .env
+
+# download docker images and initialize fpan
+cd fpan
+make init-dev
+```
+
+## Running Tests
+
+This will run all tests, assuming you've already run `make dev` to spin up the dev environment.
+
+```sh
+cd fpan-workspace/fpan
+
+make test
+```
+
+## Additional Make Commands
+
+- Run a celery auto-reloader, if you want to work on a celery task:
+  - `make autoreload-celery`
+- Run the Django shell in your terminal:
+  - `make django-shell`
+- Wipe all db and elasticsearch data. Afterwards, you'll want to run some `init-dev` command again.
+  - `make dev-down-delete-data`
+- Open a bash shell in the `arches` container to run arbitrary commands, like Django management commands and `pip install`:
+  - `make arches-bash`
+- Watch logs for `db`, `elasticsearch`, and `rabbitmq` containers.
+  - `make logs-db`, `make logs-elasticsearch`, `make logs-rabbitmq`
+
+## Configuring Pyright LSP
+
+This is bit odd. The issue is that the python environment is inside the arches docker container, but fpan is on the host machine. On Mac and Windows*, to allow the LSP to work as intended (allow for jump to definition, docs, completion), we must have a copy of the container's venv directory on the host machine, since that's where the LSP is running. This copy will only be used for its site-packages directory, which pyright is configured to inspect.
+
+> *You should only need a copy of the venv on the host on Mac and Windows machines, since both run Docker containers in virtual machines. On a Linux machine, you should be able to point pyright to the actual directory in the container, as it's really a directory on the host machine, although this is untested. To try this, edit pyrightconfig.json to look like this:
```

Comment:
I've been trying this locally on Ubuntu but can't it to work. I tried `"venvPath": "/opt"`, `"venvPath": "fpan_arches:/opt"` ssh-style, and started rooting around in `/var/lib/docker` but wasn't able to find real paths to the content. Seems like there are more pylance/vscode/container rabbit holes to go down but we'll leave it with the `docker cp` command for now. So I'll just add a commit to update this comment here. Thank you for this section though! I was wrestling with VSCode's interpreter and then realized you'd written everything I needed here.